src/rgw/rgw_crypt.cc

```diff
@@ -359,25 +359,26 @@ class AES_256_CBC : public BlockCrypt {
                      const unsigned char (&key)[AES_256_KEYSIZE],
                      bool encrypt)
   {
-    static std::atomic<bool> failed_to_get_crypto(false);
-    CryptoAccelRef crypto_accel;
-    if (! failed_to_get_crypto.load())
```

Comment:
@cbodley You are right. Once it scaled down so much it no longer is even related to all plugins.